src/tasks/components/TaskHeader.tsx

```diff
@@ -27,6 +29,12 @@ export default class TaskHeader extends PureComponent<Props> {
           <RateLimitAlert />
         </Page.Header>
         <Page.ControlBar fullWidth={true}>
+          <Page.ControlBarLeft className="task-header--cta">
+            <FeatureFlag name="flowsCTA">
```

Comment:
will wrapping the feature flag here or around the ControlBarLeft make a difference?

Reply:
not that i saw